website/docs/developer/research/research-bamboo.md

```diff
@@ -139,7 +139,7 @@ Make sure the linked repository have permissions to create plans within given pr
 :::
 
 In order to run this YAML definition from github repo, user has to setup `Repository-stored Specs` on Bamboo CI/CD [website](https://ci.earthdata.nasa.gov/build/admin/create/newSpecs.action). From the Bamboo dashboard, Choose `Specs`->`Set up Specs repository` and then select `Build project` as `ORCA` and `Repository host` as `ORCA repo`.
-The next step is to create a `Webhook` on the source repository by going to the settings option so that Bamboo knows about new commits. Webhooks allow Github repositories to communicate with Bamboo. A webhook has been created for ORCA project `https://ci.earthdata.nasa.gov/rest/api/latest/repository/scan?repositoryId=375849190` which needs to be copied to the `webhook` section in cumulus-orca repo. Furthur discussions are needed for this step with the whole team. 
+The next step is to create a `Webhook` on the source repository by going to the settings option so that Bamboo knows about new commits. Webhooks allow Github repositories to communicate with Bamboo. A webhook has been created for ORCA project `https://ci.earthdata.nasa.gov/rest/api/latest/repository/scan?repositoryId=375849190` which needs to be copied to the `webhook` section in cumulus-orca repo. Without adding webhook, Bamboo was not able to recognize the new Spec file added to the `cumulus-orca` repo. After discussion with team, a NASD ticket will need to be submitted in order for them to add the webhook. 
```

Comment:
Change NASD ticket refrence to email to NASA GitHub administrators with a link to the instruction document I sent you.

Reply:
added